v1/ao/internal/reporter/methods.go

```diff
@@ -461,7 +461,7 @@ func settingsRespStr(r *collector.SettingsResult, err error) string {
 	if r == nil {
 		return errGotNilResp.Error()
 	}
-	return fmt.Sprintf("%v %s", r.Result, r.Arg)
+	return fmt.Sprintf("%v %s %s", r.Result, r.Arg, r.Warning)
```

Comment:
Not directly related to his PR but I am wondering what will get print out for `r.Arg`

There is this piece of sensitive info - the x-tv-option signature key that might get print out (depending on how Go displays it).

Java agent explicitly mask that piece of info now: https://github.com/librato/joboe/pull/997

Reply:
The signature key is in `r.Settings.Argument`, which is not printed in the logs. `r.Arg` is a plain text and is used for the redirecting address for now.
I agree these two names are a bit confusing. :)

Reply:
Ah got it thanks!